src/mscorlib/src/System/Security/Util/StringExpressionSet.cs

```diff
@@ -33,8 +33,8 @@ internal class StringExpressionSet
         protected String[] m_expressionsArray;
 
         protected bool m_throwOnRelative;
-        
-        protected static readonly char[] m_separators = { ';' };
+
+        private const char Separator = ';';
```

Comment:
BTW: `src/mscorlib/src/System/Security/Util/...` is dead code - it will be deleted as part of the planned cleanups.